lib/constructs/provisioning-sfn-workflow.ts

```diff
@@ -56,7 +56,7 @@ export class ProvisioningWorkflow extends Construct implements IProvisioningWork
     super(scope, id);
     const { environmentName } = props;
 
-    this.provisioningJobsQueue = new AtatQueue(scope, "ProvisioningJobsQueue", { environmentName }).sqs;
+    this.provisioningJobsQueue = new FifoQueue(scope, "ProvisioningJobsQueue");
```

Comment:
Is it okay removing the `environmentName`? I know there have been some name collisions in the past with similar stack resources not being able to be deployed if one stack had a similar name.